github_pr.py

```diff
@@ -369,8 +395,16 @@ def main():
 Delete a PR
 
     github-pr delete -r dataxu/test_repo -n 17
+
+Check conditional status checks
+
+    github-pr check-condition -r dataxu/dcommand -n 84 --condition-owner-cannot-ship
+        This will check to make sure that the owner can not apply a shipable comment on their own code
+        Also available at this time is a check to make the comment is from an approved shipper
+        This check only looks at comments AFTER the latest commit, to validate that the
+        code has been most recently peer reviewed.
```

Comment:
Can you extend this example/doc a little bit? Here's an example or we could just use the below:
```
github-pr check-condition -r dataxu/dcommand -n 84 --condition-owner-cannot-ship
    This will check that a comment containing a :shipit: (the default, or other defined comment if merge_comment is set) is not from the owner of the PR, enforcing that the owner cannot give the final review or "merge go-ahead" on their own code.
github-pr check-condition -r dataxu/dcommand -n 84 --condition-approved-mergers-file
    This will check that a comment containing a :shipit: (the default, or other defined comment if merge_comment is set) comes from a user in the MAINTAINERS file in the repo's master branch.
github-pr check-condition -r dataxu/dcommand -n 84 --condition-approved-mergers "ned_flanders,marge_simpson"
    This will check that a comment containing a :shipit: (the default, or other defined comment if merge_comment is set) comes from a user in the provided list.

NB that these checks enforce that comments containing :shipit: (the default, or other defined comment if merge_comment is set) must occur AFTER the latest commit to the PR, and cannot be edited (since users with write permission can edit other users comments).
```